include/raylib-nuklear.h

```diff
@@ -52,7 +52,11 @@
 #endif  // NK_INCLUDE_FIXED_TYPES
 
 #ifndef NK_ASSERT
-#define NK_ASSERT(condition) do { if (!(condition)) { TraceLog(LOG_WARNING, "NUKLEAR: Failed assert \"%s\" (%s:%i)", #condition, "nuklear.h", __LINE__); }} while (0)
+#ifdef NDEBUG
+#define NK_ASSERT(condition) (void)0
```

Comment:
``` suggestion
#define NK_ASSERT(condition) ((void)0)
```

Reply:
Could just be replaced with `#define NK_ASSERT(condition) ;`
Should still work the same way but might not be as readable

Reply:
Interesting idea. Does that result in some compilation warnings tho? Since it ends up being extra `;`?

Reply:
I can confirm that there are no warnings from c11 and above, not sure about c99 though

Reply:
Going to keep the (void) in there. Majority of recommendations I've seen have that, rather than the empty case. Thanks for testing!